packages/trip-form/src/DateTimeSelector/index.tsx

```diff
@@ -150,8 +159,12 @@ export default function DateTimeSelector({
   const handleInputChange = (key: string) => useCallback(
     (evt: ChangeEvent<HTMLInputElement>): void => {
       handleQueryParamChange({ [key]: evt.target.value });
+      // If the user changes the time, it doesn't make sense for them to be departing now.
+      if (departArrive === "NOW") {
```

Comment:
nitpick: if departureOptions were an enum then we could reuse that instead of using a magic string

Reply:
Would you change `departureOptions` to 
```
departureOptions = {
now: "NOW"
etc
```
And then render the text strings dynamically?

Reply:
yes! but it's such a nitpick